src/python/WMCore/WMBS/MySQL/Files/SetLocationForWorkQueue.py

```diff
@@ -36,14 +37,16 @@ def execute(self, lfns, locations, isDBS=True, conn=None, transaction=None):
         Else:
         Simply insert the new locations
         """
-        binds = []
-        for lfn in lfns:
-            binds.append({'lfn': lfn})
-
         if isDBS:
-            self.dbi.processData(self.deleteSQL, binds, conn=conn,
-                                 transaction=transaction)
+            binds = []
+            for lfn in lfns:
+                binds.add({'lfn': lfn})
```

Comment:
Darn it! I saw this one failing in vocms0257, it turns out it was supposed to be an append, fixing it.